01/app.js

```diff
@@ -0,0 +1,17 @@
+// Twoim pierwszym zadaniem jest uruchomienie Webpack-a, który będzie wykonywał transpilację utworzonego kodu z JSX do JavaScript.
+
+// W pliku ./webpack.config.js (w katalogu głównym) znajdziesz gotową konfigurację Webpack-a dla React. Znajduje się tam zmienna taskNumber, która przechwowuje informacje o zadaniu, które w danym momencie przerabiasz.
+
+// Za każdym razem będziesz musiał modyfikować zawartość tej zmiennej i ponownie uruchamiać tryb developerski. Pamiętasz jak to zrobić? Gdzie zapisany jest skrót do odpowiedniej komendy?
+
+// Pamiętaj również, że musimy mieć pobrane paczki z npm, które będziemy wykorzystywać podczas pracy z React. Wiesz jak je zainstalować?
+
+// W pliku app.js musisz utworzyć kod, który przy pomocy React-a i JSX doda do elementu #root nagłówek pierwszego poziomu tj. <h1/> z napisem React działa!.
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+ 
+ReactDOM.render(
+    React.createElement('h1', null, 'React działa!'),
```

Comment:
Miałbyć jsx czyli `<h1>...</h1>` - to co napisałaś jest poprawne, ale nie jest kodem JSX :)